grakn-engine/src/main/java/ai/grakn/engine/GraknEngineServerFactory.java

```diff
@@ -66,21 +66,20 @@ public static GraknEngineServer createGraknEngineServer() throws IOException {
         CountStorage countStorage = RedisCountStorage.create(redisWrapper.getJedisPool(), metricRegistry);
         LockProvider lockProvider = new JedisLockProvider(redisWrapper.getJedisPool());
         EngineGraknTxFactory engineGraknTxFactory = EngineGraknTxFactory.create(lockProvider, config);
-        GrpcServer grpcServer = configureGrpcServer(config, engineGraknTxFactory);
-
 
         return createGraknEngineServer(engineID, service, status, metricRegistry, config, redisWrapper, indexStorage,
-                countStorage, lockProvider, runtime, httpCollaborators, engineGraknTxFactory, grpcServer);
+                countStorage, lockProvider, runtime, httpCollaborators, engineGraknTxFactory);
     }
 
     public static GraknEngineServer createGraknEngineServer(
             EngineID engineID, Service sparkService, GraknEngineStatus graknEngineStatus, MetricRegistry metricRegistry, GraknConfig graknEngineConfig,
             RedisWrapper redisWrapper, IndexStorage indexStorage, CountStorage countStorage, LockProvider lockProvider,
-            Runtime runtime, Collection<HttpController> collaborators, EngineGraknTxFactory factory, GrpcServer grpcServer) {
+            Runtime runtime, Collection<HttpController> collaborators, EngineGraknTxFactory factory) {
 
         IndexPostProcessor indexPostProcessor = IndexPostProcessor.create(lockProvider, indexStorage);
         CountPostProcessor countPostProcessor = CountPostProcessor.create(graknEngineConfig, factory, lockProvider, metricRegistry, countStorage);
         PostProcessor postProcessor = PostProcessor.create(indexPostProcessor, countPostProcessor);
+        GrpcServer grpcServer = configureGrpcServer(graknEngineConfig, factory, postProcessor);
```

Comment:
Why are we instantiating the gRPC server here instead of taking it as a parameter? I think we should keep it as a param, so that we can reuse this method also in KGMS - passing an authenticated grpc server